code/modules/clothing/spacesuits/flightsuit.dm

```diff
@@ -492,6 +520,38 @@
 		losecontrol(move = FALSE)
 	crashing = FALSE
 
+/obj/item/device/flightpack/proc/door_hit(obj/structure/mineral_door/door)
```

Comment:
These things are done this way instead of some generic function overrides for a reason, anyway ?